tests/Fixtures/inc/Engine/License/API/UserClient/getUserData.php

```diff
@@ -37,15 +37,5 @@
 		],
 		'expected' => $data,
 	],
-	'testShouldReturnDataWhenSuccess'  => [
```

Comment:
May I ask you why this fixture is deleted? It does not seem related to the code changes you applied.

Reply:
the test is too fragile, because the data on the API end changes to often. I want to do it another way, but for now I'm removing it to stop it from failing our test suites every 2 days.

Reply:
Ok. Good to know. On my last PR there are a lot of failures due to "fragile" test data related to API too. 
Like expiration data on licence. Probably this one is  related too.